src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
```

Comment:
The `Application` class is missing an access modifier. It's good practice to specify access modifiers to define the visibility of your classes explicitly. In this case, it should be `public`.